info.toml

```diff
@@ -502,3 +502,57 @@ If you're having trouble updating the distance value in the `Fish` and `Dog` imp
 2. Update the distance variable
 3. Reconstruct `self` with the updated variables (`self = MyStruct { ... }`) 
 """
+
+# MODULES
+
+[[exercises]]
+name = "modules1"
+path = "exercises/modules/modules1.cairo"
+mode = "test"
+hint = """"""
```

Comment:
If there are no hints, add "No hints this time!"

Reply:
But I would add a hint here.
- You can bring a parent's modules items in the current module with `super::item_name`

Reply:
Sweet!